tests/main/snapd-apparmor/task.yaml

```diff
@@ -0,0 +1,73 @@
+summary: Ensure snapd-apparmor works as expected
+
+environment:
+    CONSUMER_SNAP: test-snapd-policy-app-consumer
+
+debug: |
+    journalctl -u snap.apparmor.service
+
+execute: |
+    if ! systemctl is-active snapd.apparmor.service; then
```

Comment:
where is the service not active? if we don't want to run this test on i.e. systems without apparmor at all, perhaps we should instead make this a check that on systems like fedora etc that snapd.apparmor.service is not present at all?

Reply:
also we have struggled to identify what distros we expect to see snapd.apparmor.service on, and it seems we now essentially expect to find it everywhere we have apparmor, so it would be nice if this test could also essentially serve as documentation for that in that it has a comment and associated checks that we have snapd.apparmor.service on distros that should have it. Currently you have to go digging through the packaging rules to identify whether the service is included or not in a given distro